Sources/ShowAttachments.php

```diff
@@ -69,7 +69,7 @@ function showAttachment()
 	$attachTopic = isset($_REQUEST['topic']) ? (int) $_REQUEST['topic'] : 0;
 
 	// No access in strict maintenance mode or you don't have permission to see attachments.
-	if ((!empty($maintenance) && $maintenance == 2) || !allowedTo('view_attachments'))
+	if (!empty($maintenance) && $maintenance == 2)
```

Comment:
Does this show attachments in threads even without permission?

Reply:
No, this does not work here. Permissions in topics work well.